### PR TITLE
test: block-pod-exec tests, stale-images, updates-deletes tests

### PR DESCRIPTION
**Pull request by mintbomb27**
## Related Issue(s)
#546 

## Description
Contains 6 kuttl tests for other/block-pod-exec-* and block-stale-images, block-updates-deletes

## Checklist

- [x] I have read the [policy contribution guidelines](https://github.com/kyverno/policies/blob/main/README.md#contribution).
- [x] I have added test manifests and resources covering both positive and negative tests that prove this policy works as intended.
- [ ] I have added the artifacthub-pkg.yml file and have verified it is complete and correct.

### CODE REVIEW

File: other/block-stale-images/01-assert.yaml
Comment (chipzoller) [inline other/block-stale-images/01-assert.yaml:1]: Ok, this one I'm torn over. The challenge we'll have is this test will start failing once the `busybox:1.35` image reaches 6 months of age, and then we'll have to find another tag to use :) My vote is to just skip this one so it doesn't require periodic refreshing.

**Review by chipzoller — APPROVED** [review]
All look good.

File: other/block-stale-images/01-assert.yaml
Comment (mintbomb27) [reply other/block-stale-images/01-assert.yaml:1]: What if I make it to use the `:latest` tag? Hopefully it will be maintained for a couple 10 years more?

Comment (chipzoller) [reply other/block-stale-images/01-assert.yaml:1]: Yes, let's do that.

**Review by chipzoller — APPROVED** [review]

**Pull request merged**